guides/common/modules/proc_configuring-the-cleaning-unused-tasks-feature.adoc

```diff
@@ -10,26 +10,24 @@ By default, {Project} executes a cron job that cleans tasks every day at 19:45.
 * Tasks that have run successfully and are older than thirty days
 * All tasks that are older than a year
 
-.For {Project}s Upgraded from Previous Versions
-ifndef::orcharhino[]
-Until https://bugzilla.redhat.com/show_bug.cgi?id=1788615[BZ#1788615] is resolved, this functionality works only on fresh installations of {ProjectXY} and later.
-endif::[]
-If you upgrade {Project} from previous versions, this functionality is disabled by default.
-To enable {Project} to perform regular cleaning, enter the following command:
+You can configure the cleaning unused tasks feature using the below options:
 
+* To configure the time at which {Project} runs the cron job, set the `--foreman-plugin-tasks-cron-line` parameter to the time you want in cron format.
+For example, to schedule the cron job to run every day at 15:00, enter the following command:
```

Comment:
```suggestion
For example, to schedule the cron job to run every day at 15:00, run the following command:
```

Reply:
@adamlazik1 I checked the docs and it seems that we use `enter` instead of `run` everywhere in the document. I checked the style guide and both seem valid. For the sake of consistency, should we keep this as `enter`?

Reply:
Yes, sorry, this was my mistake.